rel-eng/atomic-enterprise-node.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=Atomic Enterprise Node
+After=docker.service
+After=atomic-enterprise-master.service
+After=atomic-enterprise-sdn-master.service
+After=atomic-enterprise-sdn-node.service
+Wants=docker.service
+Documentation=https://github.com/openshift/origin
+
+[Service]
+Type=notify
+EnvironmentFile=/etc/sysconfig/atomic-enterprise-node
+Environment=GOTRACEBACK=crash
+ExecStart=/usr/bin/atomic-enterprise start node --config=${CONFIG_FILE} $OPTIONS
```

Comment:
Putting it in the unit file would allow you to define the slice to match GOMAXPROCS if folks cared.